### PR TITLE
Update users.md

### PR DESCRIPTION
**Pull request by trolologuy**

### CODE REVIEW

File: docs/src/administration/users.md [hunk 1]
```diff
@@ -15,13 +15,13 @@ Any user added to a project or an environment on Platform.sh will need to [regis
...
 * **Project Administrator** - A project administrator can change settings and execute actions in any environment.
-* **Project Viewer** - A project reader can view all environments within a project but cannot execute any actions on them.
+* **Project Viewer** - A project reader can view all environments within a project through the management console but cannot execute any actions on them.
 
 A Project Viewer can have a specific role in different environments. At the environment level:
 
 * **Environment Administrator** - An environment administrator can change settings and execute actions on this environment.
 * **Environment Contributor** - An environment contributor can push code to this environment and branch the environment.
-* **Environment Viewer** - An environment reader can only view this environment.
+* **Environment Viewer** - An environment reader can only view this environment on the management console, access through SSH is not possible.
```
Comment (pjcdawkins) [inline docs/src/administration/users.md:24]: not necessarily

IMO this should link to https://docs.platform.sh/configuration/app/access.html

File: docs/src/administration/users.md [hunk 2]
```diff
@@ -15,13 +15,13 @@ Any user added to a project or an environment on Platform.sh will need to [regis
 At the project level:
 
 * **Project Administrator** - A project administrator can change settings and execute actions in any environment.
-* **Project Viewer** - A project reader can view all environments within a project but cannot execute any actions on them.
+* **Project Viewer** - A project reader can view all environments within a project through the management console but cannot execute any actions on them.
```
Comment (pjcdawkins) [inline docs/src/administration/users.md:18]: There's a slight subtlety here. It's actually not possible to be a project viewer without also having access to at least one environment.

(if you had project `viewer` and access to 0 environments, you would be automatically removed the next time the user list changes, or the next time an environment is deleted - I discovered recently)

Comment (pjcdawkins) [reply docs/src/administration/users.md:18]: And another subtlety... it's HTTP API access, which is really 'management console or the CLI or the API'.. so it's probably just best to note something about SSH access in the environment roles below, and leave this.

File: docs/src/administration/users.md (hunk 1 above)
Comment (trolologuy) [reply docs/src/administration/users.md]: I added that link in the latest commit + removed my wrong assumption to (hopefully) avoid confusion

File: docs/src/administration/users.md [hunk 3]
```diff
@@ -15,20 +15,22 @@ Any user added to a project or an environment on Platform.sh will need to [regis
...
-* **Environment Viewer** - An environment reader can only view this environment.
+* **Environment Viewer** - An environment reader can only view this environment on the management console.
 
 {{< note theme="warning" title="Important" >}}
 After a user is added to (or deleted from) an environment, it will be automatically redeployed, after which the new permissions will be fully updated.
 
 When adding users at the **project level**, however, redeployments do not occur automatically, and you will need to trigger redeployments to update those settings for each environment using the CLI command `platform redeploy`. Otherwise, user access will not be updated on those environments until after the next build and deploy commit.
 {{< /note >}}
 
+Accessing the project through SSH may differ depending on the [configuration of the project or environment](https://docs.platform.sh/configuration/app/access.html).
```
Comment (chadwcarlson) [inline docs/src/administration/users.md:32]:
```suggestion
Accessing the project through SSH may differ depending on the [configuration of the project or environment](/configuration/app/access.md).
```

File: docs/src/administration/users.md [hunk 4]
```diff
@@ -15,20 +15,22 @@ Any user added to a project or an environment on Platform.sh will need to [regis
 At the project level:
 
 * **Project Administrator** - A project administrator can change settings and execute actions in any environment.
-* **Project Viewer** - A project reader can view all environments within a project but cannot execute any actions on them.
+* **Project Viewer** - A project reader can view all environments within a project through the management console but cannot execute any actions on them.
```
Comment (chadwcarlson) [inline docs/src/administration/users.md:18]:
```suggestion
* **Project Viewer** - A project reader can view all environments within a project through the management console but cannot execute any actions on them. Note that a user must be granted access to at least one environment before they can be given project-wide viewer access.
```

**Review by chadwcarlson — APPROVED** [review]

**Pull request merged**